godot-core/src/meta/traits.rs

```diff
@@ -167,7 +167,7 @@ pub trait GodotType: GodotConvert<Via = Self> + sealed::Sealed + Sized + 'static
     message = "`Array<T>` can only store element types supported in Godot arrays (no nesting).",
     label = "has invalid element type"
 )]
-pub trait ArrayElement: ToGodot + FromGodot + sealed::Sealed + meta::ParamType {
+pub trait ArrayElement: ToGodot + FromGodot + sealed::Sealed + 'static {
```

Comment:
Note: not a breaking change because `ParamType` had `'static` as a super-bound, so we're effectively removing bounds :ok_hand: